Automattic/wp-calypso
# Login: Update the designs to match the mockups

This updates the login screens to be more inline with the mockups:

![log-in-m2-2fa](https://cloud.githubusercontent.com/assets/275961/25995062/6dc54d70-3708-11e7-9376-61ae0a243174.png)
  
#### Testing instructions
  
1. Run `git checkout fix/login` and start your server, or open a [live branch](https://delphin.live/?branch=fix/login)
2. Open the [`Login` page](http://calypso.localhost:3000/login)]
3. Log in as normal
4. Verify that the 2FA screens match the mockups, and that you can switch between an SMS, Authenticator and Mobile App option.
  
#### Additional notes
  
There are several things still to do:
- Show the phone number the SMS code is sent to
- Create a "lost phone" screen

These can be done in future PRs
  
#### Reviews
  
   
@Automattic/sdev-feed
